test/e2e/common.go

```diff
@@ -447,6 +453,10 @@ func Logf(format string, a ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, "INFO: "+format+"\n", a...)
 }
 
+func Errorf(format string, a ...interface{}) {
+	fmt.Fprintf(GinkgoWriter, "ERROR: "+format+"\n", a...)
+}
+
```

Comment:
The reason I don't want to use Expect() was because I don't think it's worth it to stop the whole process if this fetch function fails. `Errorf()` acts as a way of getting the error log without stopping the test.

Reply:
Then it is not an error. Just use `Logf` :slightly_smiling_face: